### PR TITLE
test: move InfluxQL compatibility suites so they're covered by linting & `go test`

### PR DESCRIPTION
**Pull request by danxmoran**
Around the time 2.0 went GA, 2 InfluxQL compatibility suites were ported over from Cloud. The suites were initially added within `_`-prefixed packages, in case they proved to be unstable / otherwise disruptive. The `_` prefix prevented the tests from being "seen" by the wildcards used by our linting and `go test` calls in CI.

This PR removes the `_` prefixes from the packages, so the tests are treated like any other gotest in the codebase. I think this is good because:
1. It prevents linting errors from going undetected in the code (see a few fixes in this PR)
2. It allows Circle to use its own parallelization techniques to spread the more heavyweight tests across machines
3. It allows us to simplify our Makefile and CI config

### CODE REVIEW

File: influxql/v1tests/server_helpers.go [hunk 1]
```diff
@@ -3,7 +3,6 @@ package v1tests
 import (
```
Comment (danxmoran) [inline influxql/v1tests/server_helpers.go:3]: Changes here are all from `staticcheck`

File: influxql/v1tests/server_test.go [hunk 2]
```diff
@@ -23,7 +23,6 @@ const (
 
 // Ensure the server can query with default databases (via param) and default retention policy
 func TestServer_Query_DefaultDBAndRP(t *testing.T) {
-	t.Parallel()
```
Comment (danxmoran) [inline influxql/v1tests/server_test.go:26]: For some reason, the `t.Parallel`s here were nuking Circle. Removing them didn't significantly impact the total runtime of the `test` job: the latest merge to `master` took `8m28s`, and the run from the commit that removed all these `Parallel` calls took `7m11s`

File: influxql/v1tests/server_test.go [hunk 3]
```diff
@@ -33,24 +32,24 @@ func TestServer_Query_DefaultDBAndRP(t *testing.T) {
 	}
 
 	test.addQueries([]*Query{
-		&Query{
+		{
```
Comment (danxmoran) [inline influxql/v1tests/server_test.go:35]: Cleaning up some linter warnings

File: influxql/v1tests/server_test.go [hunk 4]
```diff
@@ -403,26 +394,25 @@ func TestServer_Query_Math(t *testing.T) {
 
 // Ensure the server can handle various simple non_negative_derivative queries.
 func TestServer_Query_SelectRawNonNegativeDerivative(t *testing.T) {
-	t.Parallel()
 	s := OpenServer(t)
 	defer s.Close()
 
 	test := NewTest("db0", "rp0")
 	test.writes = Writes{
-		&Write{data: fmt.Sprintf(`cpu value=10 1278010021000000000
```
Comment (danxmoran) [inline influxql/v1tests/server_test.go:412]: staticcheck complained

**Review by davidby-influx — APPROVED** [review]
I certainly hope most of these changes were made with Find/Replace!

Looks good to me.

**Pull request merged**